content/en/scale-agent/concepts/dynamic-account-options.md

```diff
@@ -0,0 +1,56 @@
+
+Dynamic accounts accounts are are supported by the
+{{< linkWithTitle "dynamic-accounts.md" >}} endpoints. The  `zoneId` flag is used to keep each account instance unique. 
+
+> Dynamic Accounts are only available on Agents with network connectivity to the target clusters.
+
+## Configuration details
+This feature is a limited availability beta release and requires the use of custom agent and service plugins (available from your technical account manager).
+
+Dynamic Account configuration parameters are added to the Agent plugin deployment file. The required parameters must be updated as shown here:
+
+``` yaml
+- account:
+  - storage:
+    - enabled: true 
+    - kubernetes: 
+      - enabled: true
+```
+With the the configuration in place you can begin migrating and managing accounts on the Agent. Dynamic accounts can be manually or automatically managed. 
+
+### Manual configuration
+Accounts can be pulled from the Clouddriver to the cluster Agents using the {{< linkWithTitle "dynamic-accounts.md" >}}).
+
+For each account three requests are required to activate a migrated account on the Agent and set that account to `Active`:
+1. First use a `GET /armory/accounts/{accountId}` request to scan for account data. You can also use the `GET /armory/accounts/{accountId}` endpoint to source a specific account.
+2. Next, use the migrate `POST /armory/accounts` request to migrate the accounts to the Agent. This can be done as a batch process. Identified accounts are sent to the agent and set to an `INACTIVE` state.
+3. Set the migrated account to `ACTIVE` using a `PATCH /armory/accounts` request, after the migration step is completed.
+
+The initial request from the Agent to Clouddriver sets up the persistent connection, including the list of static accounts, defined in the agent configuration file `armory-agent.yml`, and by the optional `zoneId` flag. Where the flag is not applied migration still occurs. In this case accounts are polled at regular intervals and assigned to Agent pods. 
+
+Use of the `zoneId` flag is recommend for efficient migration. The `kubeconfig` file provides configuration details for connecting to the target clusters that are sent as part in a request. 
+
+If the agent is not able to connect to the target clusters, the Clouddriver sends the `SaveAccounts` operation to an Agent with a unique zone identifier. On demand HTTP requests indicate what accounts are removed from Agents and managed by Clouddriver.
+
+#### Interceptor option
+An interceptor option is provided to ease the configuration steps. With this option the configuration steps are minimized and the new account is enabled with only one request to set it to `ACTIVE` from the initial `INACTIVE` state.
```

Comment:
I would say "to ease the onboarding/migration steps". 
"With this option the configuration steps are minimized for newly added accounts using the  **POST** `/credentials` and the new account **can be** enabled with only one request to set it to `ACTIVE` from the initial `INACTIVE` state."